manon/visually-hidden.scss

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------*/
+/*-------------- components/visually-hidden.scss -----------------*/
+/*---------------------------------------------------------------*/
+.visually-hidden {
+    clip-path: polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px);
+    white-space: nowrap;
+    overflow: hidden;
+    width: 0;
+    height: 0;
+    max-width: 0;
+    max-height: 0;
+
+    /* Overwriting any specifics that might have been set.    */
+    /* !important is used here to solve any specificity issues */
```

Comment:
Thanks for the thorough comment. This is imo just what !important should be used for, but a little explanation on this is great for future reference or other devs that will come across this.